### DISCUSSION TITLE
WhatsApp

### DISCUSSION THREAD

**Discussion by gumbarros** [fonoster/yaps#332] [Ideas]
Is your feature request related to a problem? Please describe.
I'm dependent on using Twilio because of the WhatsApp API integration.

Describe the solution you'd like
A WhatsApp integration to fonoster, like the Twilio one.

Describe alternatives you've considered
Twilio.

Additional context
It will be good to have a FOSS WhatsApp API alternative.

**Comment by gumbarros** [comment]
Clarifing the discussion, in my country WhatsApp is the number one app for messaging, any person here, uses WhatsApp for anything. A FOSS API, would be greatful.

[Current Twilio WhatsApp API](https://www.twilio.com/whatsapp)

**Reply by ilomon10** [reply]
check this [repo](https://github.com/WhiskeySockets/Baileys), and make 3rd party software to integrate with fonoster
